html/Turn of Tides Food Ingredient Additions

```diff
@@ -0,0 +1,89 @@
+food = {
+	
+	//DST Turn Of Tides Ingredient Additions	
+	kelp: {
+				name: 'Kelp Fronds',
+				uncookable: true,
+				isveggie: true,
+				health: -healing_tiny,
+				hunger: calories_tiny,
+				sanity: -sanity_small,
+				perish: perish_med,
+				stack: stack_size_smallitem,
+				dry: 'kelp_dried',
+				drytime: dry_superfast
+				mode: 'together'
+			},
+			
+	kelp_cooked: {
+				name: 'Cooked Kelp Fronds',
+				uncookable: true,
+				isveggie: true,
+				veggie: 1,
+				health: 0,
+				hunger: calories_tiny,
+				sanity: -sanity_tiny,
+				perish: perish_med,
+				stack: stack_size_smallitem,
+				mode: 'together'
+			},
+	
+	kelp_dried: {
+				name: 'Dried Kelp Fronds',
+				uncookable: true,
+				isveggie: true,
+				veggie: 1,
+				health: healing_tiny,
+				hunger: calories_tiny,
+				sanity: sanity_small,
+				perish: perish_preserved,
+				stack: stack_size_smallitem,
+				mode: 'together'
+			},
+
+	moonbutterflywings: {
+				name: 'Moon Moth Wings',
+				isveggie: true,
+				decoration: 2,
```

Comment:
I added Moon Moth Wings in my fork and was wondering if they are decoration. Did you check or assume based on Butterfly Wings? Where could I find that tag anyway? I looked at the .lua files and there was no mention of decoration there.

Reply:
These are the key files to look at:

1. cooking.lua lists the ingredient values for foods, which is used for the crockpot (located in the scripts folder). For Moon Moth Wings, it looks like such:
AddIngredientValues({"moonbutterflywings"}, {decoration=2})

2. prepareparedfoods.lua lists the crockpot recipes for foods that can be made in a crockpot,
while preparedfoods_warly.lua lists recipes that can be made only be Warly. Both are located in the scripts folder.

3. veggies.lua (inside the prefabs folder) will list the hunger, health, and sanity values. Foods listed in here are treated also as a Foodtype Veggie, with foodtypes being used to set diets for characters and mobs. 

For stuff not listed in any of these files, you typically have to look inside the lua file for that prefab name. It will typically list the hunger, health, sanity, stack size, and foodtype in there.

Reply:
Thanks, I'll check those files! I've only checked the specific ones for the Butterfly/Moon Moth Wings. I've also seen there's carrot.lua so it looked like all prefabs should have their own file with all info. Although I did suspect there was a list of just recipes, just haven't looked for it xD

Reply:
The majority of the crop foods don't have their own prefab files, so you're most likely going to have top refer to veggies.lua for their values (the first line is their raw values, 2nd is cooked values, and third is dry values where applicable, such as Kelp). 

The amount of digging you'll have to do will depend on the item honestly.